<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Instrument Control Homepage </string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 230, 230);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>16</number>
    </property>
    <property name="rightMargin">
     <number>16</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <property name="spacing">
     <number>20</number>
    </property>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-color: rgb(100, 100, 100);
	font: 20pt &quot;MS Shell Dlg 2&quot;;
	border-width: 2px;        
	border-style: outset;
	border-radius: 16px;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover{
	background-color: rgb(225, 240, 255);
}
QPushButton:pressed{
	background-color: rgb(245, 245, 245);
}
</string>
      </property>
      <property name="text">
       <string>Keysight
Dual Channel
Attenuator</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-color: rgb(100, 100, 100);
	font: 20pt &quot;MS Shell Dlg 2&quot;;
	border-width: 2px;        
	border-style: outset;
	border-radius: 16px;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover{
	background-color: rgb(225, 240, 255);
}
QPushButton:pressed{
	background-color: rgb(245, 245, 245);
}
</string>
      </property>
      <property name="text">
       <string>FiberLabs
EDFA</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-color: rgb(100, 100, 100);
	font: 20pt &quot;MS Shell Dlg 2&quot;;
	border-width: 2px;        
	border-style: outset;
	border-radius: 16px;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover{
	background-color: rgb(225, 240, 255);
}
QPushButton:pressed{
	background-color: rgb(245, 245, 245);
}
</string>
      </property>
      <property name="text">
       <string>LightWaves2020
EDFA</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-color: rgb(100, 100, 100);
	font: 20pt &quot;MS Shell Dlg 2&quot;;
	border-width: 2px;        
	border-style: outset;
	border-radius: 16px;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover{
	background-color: rgb(225, 240, 255);
}
QPushButton:pressed{
	background-color: rgb(245, 245, 245);
}
</string>
      </property>
      <property name="text">
       <string>Keysight
Attenuator
/Power Meter</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-color: rgb(100, 100, 100);
	font: 20pt &quot;MS Shell Dlg 2&quot;;
	border-width: 2px;        
	border-style: outset;
	border-radius: 16px;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover{
	background-color: rgb(225, 240, 255);
}
QPushButton:pressed{
	background-color: rgb(245, 245, 245);
}
</string>
      </property>
      <property name="text">
       <string>TLG-300
Tunable Laser</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-color: rgb(100, 100, 100);
	font: 20pt &quot;MS Shell Dlg 2&quot;;
	border-width: 2px;        
	border-style: outset;
	border-radius: 16px;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover{
	background-color: rgb(225, 240, 255);
}
QPushButton:pressed{
	background-color: rgb(245, 245, 245);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Instrument Control</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
